build-all.sh

```diff
@@ -16,7 +15,8 @@ goplat=( darwin windows linux )
 # Supported CPU architectures: amd64
 goarc=( amd64 )
 # Supported database tags
-dbtags=( mysql mongodb rethinkdb )
+dbadapters=( mysql mongodb rethinkdb )
+dbtags=( ${dbadapters[@]} alldbs )
```

Comment:
What's the value of splitting it into two vars, why not like you did before `( mysql mongodb rethinkdb alldbs )`?

Reply:
I did it so it's easy to concatenate all available db adapters.